pele/gui/ui/mplwidget.py

```diff
@@ -1,6 +1,6 @@
 from PyQt4 import QtGui, QtCore
 from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg as FigureCanvas
-from matplotlib.backends.backend_qt4agg import NavigationToolbar2QTAgg as NavigationToolbar
+from matplotlib.backends.backend_qt4agg import NavigationToolbar2QT as NavigationToolbar
```

Comment:
ok.  matplotlib changed backwards compatibility?  Does the new one work on older versions of matplotlib?  e.g. the version installed through ubuntu apt-get?